src/components/UpdatePaymentDetailsPopup/UpdatePaymentDetailsPopup.js

```diff
@@ -77,14 +77,17 @@ const UpdatePaymentDetailsPopup = ({ updatePaymentDetailsSection }) => {
   const { t } = useTranslation();
   const [dropInInstance, setDropInInstance] = useState(null);
   const { paymentMethods } = useSelector(state => state.publisherConfig);
-  const { paymentDetails } = useSelector(state => state.paymentInfo);
+  const { paymentDetails } = useSelector(state => state.paymentDetails);
   const { selectedPaymentMethod } = useSelector(state => state.paymentMethods);
   const { loading: isFinalizeAddPaymentDetailsLoading } = useSelector(
     state => state.finalizeAddPaymentDetails
   );
   const [isActionHandlingProcessing, setIsActionHandlingProcessing] = useState(
     false
   );
+  const updatePaymentDetailsSection = () => {
```

Comment:
I see that this updatePaymentDetailsSection is also passed down to DeletePaymentMethod and FinalizeAddPaymentDetails, could we also get rid of this prop? Simply use dispatch directly in DeletePaymentMethod and FinalizeAddPaymentDetails

Reply:
great that we are not passing is down! is it still needed here then? If not, please delete it :)